src/pptrees/prefix_graph.py

```diff
@@ -530,6 +533,13 @@ def hdl(self,out=None,full_flat=False):
         if out is not None:
             with open(out,'w') as f:
                 print(hdl,file=f)
+            # Copy mapping file from package to local directory
+            with importlib.resources.path("pptrees","mappings") as pkg_map_dir:
+                pkg_map_file = pkg_map_dir / (mapping+'_map.v')
+            local_map_file = os.path.join(os.path.dirname(os.path.abspath(out)))
```

Comment:
you might want to indent that under with to make it clearer that they are all semantically related to `pkg_map_dir`